app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".ProfileActivity"
+            android:exported="false" />
```

Comment:
You may add a go back button (I think they call it upward navigation), in order to make the app demo easier.
Actually the go back button of the phone (emulator) works too so this is not much of an issue.

In any case navigation will pretty much change later on in the project so it is not a big deal.